### PR TITLE
refactor: explicitly mention imported symbols

### PR DESCRIPTION
**Pull request by damil**
### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

**Review by alexgarel — COMMENTED** [review]
LGTM.

@stephanegigandet I would like you approval !

File: tests/unit/expected_test_results/taxonomies/eurocodes.csv [hunk 1]
```diff
@@ -1,167 +1 @@
 eurocode_2_group_2	eurocode_2_group_3	ingredient
-7.10	7.10.10	en:split-pea
```
Comment (stephanegigandet) [inline tests/unit/expected_test_results/taxonomies/eurocodes.csv:2]: Why do we have this change?

File: lib/ProductOpener/Ingredients.pm [hunk 2]
```diff
@@ -57,9 +57,7 @@ BEGIN {
 	@EXPORT_OK = qw(
 
 		&parse_ingredients_text_service
-		&extend_ingredients_service
```
[the remark this answers is not in the document]
Comment (alexgarel) [reply lib/ProductOpener/Ingredients.pm:60]: fixed.

**Review by stephanegigandet — APPROVED** [review]
Looks good, thank you very much @damil and @alexgarel !

**Comment by smonff** [comment]
This is super useful for the [object module experiment](!10007). It makes the refactoring much easier and safer.

**Pull request merged**